filepathSlugs.json

```diff
@@ -2788,6 +2829,67 @@
     "response-balance-too-low",
     "response-transport-error"
   ],
+  "src/pages/atomicdex/api/v20-dev/non_fungible_tokens/get_nft_list/index.mdx": [
+    "get-a-list-of-nfts",
+    "request-parameters",
+    "response-parameters",
+    "example-with-no-optional-params",
+    "example-with-optional-limit-and-page-number-params",
+    "example-with-optional-spam-protection",
+    "error-responses",
+    "unsupported-chain-type"
+  ],
+  "src/pages/atomicdex/api/v20-dev/non_fungible_tokens/get_nft_metadata/index.mdx": [
+    "get-nft-metadata",
+    "request-parameters",
+    "response-parameters",
+    "example",
+    "error-responses"
+  ],
+  "src/pages/atomicdex/api/v20-dev/non_fungible_tokens/get_nft_transfers/index.mdx": [
+    "get-a-list-of-nft-transfers",
+    "request-parameters",
+    "response-parameters",
+    "example-with-date-and-send-filters"
+  ],
+  "src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx": [
+    "non-fungible-tokens-nfts",
+    "nft-information-methods",
+    "nft-transaction-methods"
+  ],
+  "src/pages/atomicdex/api/v20-dev/non_fungible_tokens/query_nft_database_tables/index.mdx": [
+    "query-nft-database-tables",
+    "nft-list-tables",
+    "example-coin-nft-list-table-query",
+    "nft-transfer-table",
+    "example-scanned-nft-blocks-table-query",
```

Comment:
should be `"example-nft-transfer-table-query"` or smth like this, but not scanned-nft-blocks, as above is nft transfer table.


Also there is some problem with sql command display
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/ed7efa77-bb08-4822-b580-06b26840104e)

I mean example coin_nft_list table query looks differently
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/3ded9cf8-a2d0-4d07-9ddc-6bff5e5aabb8)

Reply:
these slugs are automatically generated for internal anchor links, so I wont mess with it.

Reply:
fixed sql command formats in [`3cd7280` (#42)](https://github.com/KomodoPlatform/komodo-docs-mdx/pull/42/commits/3cd72803054e40e352df7d7f452816e539842e10)

Reply:
@laruh @smk762 can you please explain what the issue with the autogenerated slugs is?

Reply:
I get it, the headers are repeating. Need to change one of them. fixed in [`e0267a3` (#42)](https://github.com/KomodoPlatform/komodo-docs-mdx/pull/42/commits/e0267a30638996e8260d3c5b5032285619c4562e)